CovertInfiltration/Localization/CovertInfiltration.int

```diff
@@ -70,6 +70,9 @@ strInfiltrating=Infiltrating
 strOnCovertAction=On Covert Actions
 strUnavailable=Unavailable
 
+strInfilLabel=INFILTRATION HOURS
+strDeterLabel=DETERRENCE PERCENT

```

Comment:
Maybe "risk reduction percent" is clearer?

Reply:
We use "deterrence" in most other UI mentioning it though

Reply:
Hmm, only in tutorial text, not in any actual UI